undertow/src/main/java/org/wildfly/extension/undertow/deployment/UndertowDeploymentProcessor.java

```diff
@@ -25,6 +25,7 @@
 import static org.jboss.as.ee.component.Attachments.STARTUP_COUNTDOWN;
 import static org.jboss.as.server.security.SecurityMetaData.ATTACHMENT_KEY;
 import static org.jboss.as.server.security.VirtualDomainMarkerUtility.isVirtualDomainRequired;
+import static org.jboss.as.web.common.VirtualHttpServerMechanismFactoryMarkerUtility.isVirtualMechanismFactoryRequired;
```

Comment:
Not now but in the future I wonder if we could convert both forms of "virual" into a common approach so Undertow DUPs don't need to know the details.